package.json

```diff
@@ -27,12 +27,13 @@
     "prepare:drive:mobile": "(cd src/drive/targets/mobile && rm -rf platforms && rm -rf plugins && cordova prepare)",
     "run:drive:android": "(cd src/drive/targets/mobile && cordova run android --device)",
     "run:drive:android:emulator": "(cd src/drive/targets/mobile && cordova run android --emulator)",
-    "build:drive:android": "npm run build:drive:mobile && (cd src/drive/targets/mobile && cordova build android --release)",
-    "build:drive:android:debug": "npm run build:drive:mobile && (cd src/drive/targets/mobile && cordova build android --debug)",
-    "sign:drive:android": "(cd src/drive/targets/mobile && apksigner sign --ks keys/android/cozy-drive-release-key.jks --out build/android/cozy-drive.apk platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk)",
+    "build:drive:android": "(cd src/drive/targets/mobile && cordova build android --release)",
+    "build:drive:android:debug": "(cd src/drive/targets/mobile && cordova build android --debug)",
+    "sign:drive:android": "(cd src/drive/targets/mobile && apksigner sign --ks keys/android/cozy-drive-release-key.jks --ks-pass env:ANDROID_KEYSTORE_PASS --out build/android/cozy-drive.apk platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk)",
```

Comment:
Does that mean we need to create a new ENV on our local machine?

Reply:
We still need to sign to publish ?

Reply:
Right, in my head we would download the app from downcloud and publish manually, but it's betteer with the cli. I'll add a temporary second command.